src/hooks/useAccountRiskCheck.ts

```diff
@@ -11,11 +11,14 @@ export default function useAccountRiskCheck(account: string | null | undefined)
       const riskCheckLocalStorageKey = `risk-check-${account}`
       const now = Date.now()
       try {
+        // Check local browser cache
         const storedTime = localStorage.getItem(riskCheckLocalStorageKey)
         const checkExpirationTime = storedTime ? parseInt(storedTime) : now - 1
         if (checkExpirationTime < Date.now()) {
-          const headers = new Headers({ 'Content-Type': 'application/json' })
-          fetch('https://screening-worker.uniswap.workers.dev', {
+          const headers = new Headers({
+            'Content-Type': 'application/json',
+          })
```

Comment:
odd that this formatting is changing

Reply:
you might want to check that your editor is correctly configured to use our lint/prettier setup